core/3rdPartyLibs/AL/EFX-Util.h

```diff
@@ -420,3 +420,22 @@ void ConvertOcclusionParameters(EAXOCCLUSIONPROPERTIES *pOcProp, EFXLOWPASSFILTE
 	{ 26,  19.6f,	0.840f,	-1000,  -500,	0,		4.62f,	0.64f,	1.23f,  -700,	0.032f,	0.00f,0.00f,0.00f,	-200,	0.049f,		0.00f,0.00f,0.00f,	0.250f, 0.000f, 0.250f, 0.110f, -5.0f,  5000.0f,	250.0f, 0.00f,	0x3f }
 #define REVERB_PRESET_SMALLWATERROOM \
 	{ 26,  36.2f,	0.700f, -1000,  -698,   0,		1.51f,	1.25f,	1.14f,  -100,	0.020f, 0.00f,0.00f,0.00f,	300,	0.030f,		0.00f,0.00f,0.00f,	0.179f, 0.150f, 0.895f, 0.190f, -7.0f,  5000.0f,	250.0f, 0.00f, 0x0 }
+
+/* Reverb Flags */
+
+// Reverberation decay time
+constexpr unsigned long REVERB_FLAG_DECAY_TIME_SCALE = 0x1;
+// Reflection level
+constexpr unsigned long REVERB_FLAG_REFLECTIONS_SCALE = 0x2;
+// Initial reflection delay time
+constexpr unsigned long REVERB_FLAG_REFLECTIONS_DELAY_SCALE = 0x4;
+// Reverberation level
+constexpr unsigned long REVERB_FLAG_REVERB_SCALE = 0x8;
+// Late reverberation delay time
+constexpr unsigned long REVERB_FLAG_REVERB_DELAY_SCALE = 0x10;
+// Echo time
+constexpr unsigned long REVERB_FLAG_ECHO_TIME_SCALE = 0x40;
+// Modulation time
+constexpr unsigned long REVERB_FLAG_MODULATION_TIME_SCALE = 0x80;
+// Limit unnaturally long decay times of high-frequency sounds by forcing a limit to the decay time to be calculated from the Air Absorption HF value
+constexpr unsigned long REVERB_FLAG_DECAY_HF_LIMIT = 0x20;
```

Comment:
Where did these constants come from?

This header file comes from OpenAL originally (note the "3rd party" in the path), and I'm fairly certain the original is a C-compatible header, meaning it wouldn't use `constexpr`.  I'd prefer not to customize a third party header, so this should either be updated to match a version of the OpenAL header, or be put into another appropriate header file.

Reply:
Ah, right. My bad, that's a hacky copy-paste from another OpenAL file. I forgot to fix this before the PR, will do in the next commit.

Reply:
Okay, after more digging, those flags aren't defined in OpenAL at all. They come from a file named `eax.h` which *appears* to be from the original DirectSound3D library - but I can't find where to download it anymore. The `eax.h` file is redistributed in various GitHub projects, though. Most notably [doom3-gpl](https://dev.visucore.com/doom3/doxygen/eax_8h_source.html) (where it's GPL-v3 licensed) and [GTA RE3](https://git.pocketdevs.org/Substitute/Re3/src/branch/master/src/audio/eax/eax.h) (unlicensed).

We could add `eax.h` as a new third-party file, although I'm not really fond of the idea because of the redundant types it declares and the need to fix the includes. Or we could just keep the flags we're interested in and put them in another header, whether an existing one like `plEAXListenerMod.h`, or a new header somewhere (in that case, do you have a preference for the location/naming of the file ?).

I'll probably squash/force push some of the commits afterwards in order to remove my fiddlings from the history of EFX-Util.h.